tests/admin_views/admin.py

```diff
@@ -409,7 +409,13 @@ def redirect_to(modeladmin, request, selected):
 
 @admin.action(description="Download subscription")
 def download(modeladmin, request, selected):
-    buf = StringIO("This is the content of the file")
+    from django.db.models.query import QuerySet
+
+    if isinstance(selected, QuerySet):
```

Comment:
🎗️ This handle should be documented? Or should the action get `modeladmin, request, selected, object` as params? With `selected` as `None` when is change view or `object` as None when is list view 🤔